README.md

```diff
@@ -15,6 +15,31 @@ Current status:
 
 See the [docs](https://github.com/MastodonC/witan.models.demography/blob/master/doc/intro.md) for more info about the methodology.
 
+## Development tools
+
+The `:dev` leiningen profile includes a dependency to the library `witan.workspace-executor`.
+This library contains tool that are useful for developer contributing to `witan.models.demography`.
+
+* visualise your model workflow:
+
+You need to have `Graphviz` installed:
+
+-> Ubuntu: `> sudo apt-get install graphviz`
+
+-> MacOS: `> brew install graphviz`
+
+For any OS you should also be able to install it with Pip: `pip install graphviz`.
+
+```Clojure
+(witan.workspace-executor/view-workkflow my-workflow)
+```
+
+* print logs to your repl for debugging:
+
+```Clojure
+(witan.workspace-executor/set-api-logging! println)
+```
+
 ## Requirements
```

Comment:
Maybe it would be more suitable to stick all of this 
```You need to have `Graphviz` installed:
+
+-> Ubuntu: `> sudo apt-get install graphviz`
+
+-> MacOS: `> brew install graphviz`
+
+For any OS you should also be able to install it with Pip: `pip install graphviz`.```

In the requirements instead?